### ISSUE TITLE
508 compliance issues (Accessibility)  Table row elements on both the API Details and Reports pages are missing the necessary html "role=row" accessibility attributes.

### ISSUE THREAD

**Issue by christopherwoodland** [Azure/api-management-developer-portal#1387]
## Bug description

Our client has noticed the following accessibility concerns and asked that we submit these concerns to the product team for resolution. These issues violate the 508 compliance, which the customer is bound to.  I am willing to aid in any way to bring these concerns to resolution. Please let me know if additional context or information is required to help move these concerns along.  I can be reached internally via teams or email cwoodland@microsoft.com
Thank you in advance!

1. Table row elements on both the API Details and Reports pages are missing the necessary html "role=row" accessibility attributes.
![image](https://user-images.githubusercontent.com/16299417/125957421-1fb51176-2e4d-4fc5-a737-de11ca200572.png)

#1383
#1384
#1385
#1386

**Comment by jdm4344** [comment]
Hi @christopherwoodland,

Apologies, but each of these 508 concerns were added already as the following issues: #1374, #1375, #1378, #1379, #1380. Guess we had some miscommunication going on. I'm sure the product team would prefer less clutter, so one of us could close our set of issues and reference them in the ones that we keep open.  I was able to provide a little more detail about what was found in our testing, but I'll leave the decision up to you, just let me know. Thanks.

**Comment by christopherwoodland** [comment]
@jdm4344  Perfect!  I will reference and close these issues.

#1374, #1375, #1378, #1379, #1380.
